lib/node_modules/@stdlib/blas/ext/base/dapxsumkbn/manifest.json

```diff
@@ -34,7 +34,12 @@
 				"-lm"
 			],
 			"libpath": [],
-			"dependencies": []
+			"dependencies": [
+				"stdlib/napi/export.h",
+				"stdlib/napi/argv.h",
+				"stdlib/napi/argv_int64.h",
+				"stdlib/napi/argv_strided_float64array.h"
+			]
```

Comment:
This whole file is TAB indented, can you please fix this?

Reply:
Thanks for the review. I fixed everything you suggested.